django_q/cluster.py

```diff
@@ -231,7 +232,10 @@ def reincarnate(self, process):
             )
         else:
             if prometheus_multiprocess:
-                prometheus_multiprocess.mark_process_dead(process.pid)
+                prometheus_path = os.getenv("PROMETHEUS_MULTIPROC_DIR")
```

Comment:
Implementation looks good. Thanks!

I am a nitpicking here, but could you make this a one liner (so both conditions in one if statement) and then add a comment to indicate that we check this to make sure the library is configured?

Reply:
Done

Reply:
Thanks! I just noticed that there is also a lower case environment variable (same name) that gets accepted to, so I have added that one too. Didn't want to bother you again!
I will get this merged.